docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -161,12 +161,16 @@ Some cloud providers require you to use their PCCS.
 - Alibaba Cloud: See the [Alibaba Cloud documentation] for details on configuring the quote provider. The
   documentation shows the required `sgx_default_qcnl.conf` changes.
 
+- IBM Cloud: See the [IBM Cloud documentation] for details on configuring the quote provider. The
```

Comment:
This is just for the VM instances, right? 

I assume the bare metal instances don't need these changes to the config.